src/Benchmarks/link-a-thon.api-template.json

```diff
@@ -40,5 +40,30 @@
       "Project": "src/PlatformJson/PlatformJson.csproj"
     },
     "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true", "/p:UseStaticHost=true", "/p:UsePublishFilterList=true"]
+  },
+
+  "LinkAThonGrpcBaseline": {
+    "BuildArguments": [ ],
+    "Client": "Grpc",
+    "ClientProperties": {
+      "Scenario": "Unary",
+      "GrpcClientType": "GrpcNetClient"
+    }
+  },
+  "LinkAThonGrpcTrimmedAndR2R": {
+    "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true" ],
+    "Client": "Grpc",
+    "ClientProperties": {
+      "Scenario": "Unary",
+      "GrpcClientType": "GrpcNetClient"
+    }
+  },
+  "LinkAThonGrpcTrimmedAndR2RSingleFile": {
+    "BuildArguments": [ "/p:PublishTrimmed=true", "/p:LinkAggressively=true", "/p:PublishReadyToRun=true", "/p:UseStaticHost=true"],
```

Comment:
very cool and very legal - does this work?

Reply:
I don't know. Merge and find out?!!?!

Reply:
You can also just do a run to test it without merging the file? It should be easy to test in the benchmarking environment.

Reply:
This scenario I can't because `/p:UseStaticHost=true`, but I will check the baseline gRPC scenario works.

Reply:
Or you connect to the VPN and you start the job on our servers to find out

Reply:
I mean you should be able to run it on the benchmark server.